src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -2020,9 +2020,10 @@ def process_set_applications_namespace(cmd, namespace):  # pylint: disable=unuse
 
 def process_gallery_image_version_namespace(cmd, namespace):
     from azure.cli.core.azclierror import InvalidArgumentValueError
-    TargetRegion, EncryptionImages, OSDiskImageEncryption, DataDiskImageEncryption, ConfidentialVMEncryptionType = \
-        cmd.get_models('TargetRegion', 'EncryptionImages', 'OSDiskImageEncryption', 'DataDiskImageEncryption',
-                       'ConfidentialVMEncryptionType')
+    TargetRegion, EncryptionImages, OSDiskImageEncryption, DataDiskImageEncryption, ConfidentialVMEncryptionType, \
+    GalleryTargetExtendedLocation, GalleryExtendedLocation = cmd.get_models(
+        'TargetRegion', 'EncryptionImages', 'OSDiskImageEncryption', 'DataDiskImageEncryption',
+        'ConfidentialVMEncryptionType', 'GalleryTargetExtendedLocation', 'GalleryExtendedLocation')
```

Comment:
Please upgrade the version of `galleries` to `2022-01-03` in `_shared.py`, and then obtain the `GalleryTargetExtendedLocation` and `GalleryExtendedLocation` modules by  `cmd.get_models('GalleryTargetExtendedLocation', 'GalleryExtendedLocation', operation_group='galleries')`